inference-engine/samples/classification_sample_async/main.cpp

```diff
@@ -104,28 +104,22 @@ int main(int argc, char* argv[]) {
         // -------- Step 3. Apply preprocessing --------
         const ov::Layout tensor_layout{"NHWC"};
 
-        // clang-format off
-        model = PrePostProcessor().
-            // 1) InputInfo() with no args assumes a model has a single input
-            input(InputInfo().
-                // 2) Set input tensor information:
-                // - precision of tensor is supposed to be 'u8'
-                // - layout of data is 'NHWC'
-                tensor(InputTensorInfo().
-                    set_element_type(ov::element::u8).
-                    set_layout(tensor_layout)).
-                // 3) Here we suppose model has 'NCHW' layout for input
-                network(InputNetworkInfo().
-                    set_layout("NCHW"))).
-            output(OutputInfo().
-                // 4) Set output tensor information:
-                // - precision of tensor is supposed to be 'f32'
-                tensor(OutputTensorInfo().
-                    set_element_type(ov::element::f32))).
-            // 5) Once the build() method is called, the preprocessing steps
-            // for layout and precision conversions are inserted automatically
-        build(model);
-        // clang-format on
+        PrePostProcessor proc(model);
```

Comment:
@vladimir-dudnik This is how API may look like, can you please provide your feedback. It is not 'chaining' style and doesn't require `std::move`s but it probably looks more straightforward